Makefile

```diff
@@ -1,6 +1,8 @@
 DIST := dist
 IMPORT := code.gitea.io/gitea
 
+export PATH := $(go env GOPATH)/bin:$(PATH)
```

Comment:
A very tiny nit, could you use `$(GO)` in lieu of `go`. You may need to move this declaration down a few lines below where `GO` is defined.

Reply:
Done